packages/lpa-questionnaire-web-app/src/views/upload-plans.njk

```diff
@@ -0,0 +1,115 @@
+{%- from "govuk/components/file-upload/macro.njk" import govukFileUpload -%}
+{%- from "moj/components/multi-file-upload/macro.njk" import mojMultiFileUpload -%}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+{% block beforeContent %}
+  {{ govukBackLink({
+    text: 'Back',
+    href: backLink,
+    attributes: {
+      'data-cy': 'back'
+    }
+  }) }}
+{% endblock %}
+
+{% block pageTitle %}
+  Upload plans used to reach the decision - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK
+{% endblock %}
+
+{% block content %}
+
+  {{ govukErrorSummary({
+    titleText: 'There is a problem',
+    classes: not errors and 'moj-hidden',
+    errorList: errorSummary
+  }) }}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <form action="" method="POST" novalidate>
+        <span class="govuk-caption-l"><span class="govuk-visually-hidden">Section. </span>Required documents</span>
+        <h1 class="govuk-heading-l">Upload the plans used to reach the decision</h1>
+        {{ govukInsetText({
+          text: "File size should be no more than FILESIZE" | replace("FILESIZE", maxFileSize)
+        }) }}
+
+        {% set uploadHtml %}
+          {{ govukFileUpload({
+            id: 'plans',
+            name: 'plans',
+            classes: 'moj-multi-file-upload__input',
+            label: {
+              text: 'Upload a file',
+              classes: 'govuk-label--m'
+            },
+            errorMessage: errorMessage
+          }) }}
+
+          {{govukButton({
+            text: 'Upload file',
+            classes: 'govuk-button--secondary moj-multi-file-upload__button'
+          })}}
+        {% endset %}
+
+        {{ mojMultiFileUpload({
+          uploadedFiles: {
+            heading: { text: 'Files added' },
+            items: uploadedFiles
+          },
+          uploadHtml: uploadHtml
+        }) }}
+
+        {% set acceptedFormats %}
+          <ul class="govuk-list govuk-list--bullets">
+            <li>PDF – <strong>.pdf</strong></li>
+            <li>Microsoft Word – <strong>.doc</strong> or <strong>.docx</strong></li>
+            <li>TIF – <strong>.tif</strong> or <strong>.tiff</strong></li>
+            <li>JPEG – <strong>.jpg</strong> or <strong>.jpeg</strong></li>
+            <li>PNG – <strong>.png</strong></li>
+          </ul>
+        {% endset %}
+
+        {{ govukDetails({
+          summaryText: 'Accepted file formats',
+          html: acceptedFormats
+        }) }}
+
+        {{ govukButton({
+          text: 'Save and continue'
+        }) }}
+      </form>
+    </div>
+
+    {{
+      appealDetails({
+        appeal: appeal
+      })
+    }}
+  </div>
+{% endblock %}
+
+{% block pageScripts %}
+<script src="/assets/moj/all.js"></script>
+<script
+  src="https://code.jquery.com/jquery-3.5.1.min.js"
```

Comment:
What's the advantage of doing it this way over the CDN? CDN has good chance for caching etc.

Reply:
Good Q - so from my pov advantages would be:

* calling out the dependency explicitly in a central known place (package json)
* having it in package json means it gets audited (npm audit)
* one place to update if needed, rather than per page 
* it follows the existing convention of other third party assets - moj, govuk

downsides:

* having as a first party dep means it is going to be loaded on every page, unless clever code splitting is implemented  
* your way = potentially already cached in users browser
* only impacts one page so why over complicate

All things considered, I'm happy to :+1: it if no one has any stronger opinions either way.

Reply:
I think we can avoid the issue of it being on every page - for example the GovUK FE JS wasn't running on forms app as it wasn't declared in the scripts at all. We still have to include the file so still have the level of control.

Reply:
Made the decision to include jquery locally. I think overall the better solution.